index.html

```diff
@@ -2362,9 +2363,7 @@
     window.addEventListener('vrdisplayconnect', _updateDisplays);
     window.addEventListener('vrdisplaydisconnect', _updateDisplays);
 
-    const audioCtx = new AudioContext({
-      sampleRate: 48000,
-    });
+    const audioCtx = new AudioContext();
```

Comment:
Posterity: the bug this might potentially fix is a crash when the audio hardware does not support this rate.